packages/material-ui-unstyled/src/composeClasses/composeClasses.d.ts

```diff
@@ -0,0 +1,5 @@
+export default function composeClasses(options: {
+  slots: Record<string, string[]>;
+  classes: Record<string, string>;
+  getUtilityClass: (slot: string) => string;
+}): Record<string, string>;
```

Comment:
`classes` is optional, and if we want to give a bit more structure, I would expect the following to be true: 

```suggestion
export default function composeClasses<T extends keyof any>(options: {
  slots: Record<T, string[]>;
  classes?: Record<string, string>;
  getUtilityClass: (slot: string) => string;
}): Record<T, string>;
```

[playground](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBAE2AMwIYFcA29noHYDGMAlhHnARALaQDOwAwpqrfbQDwAqcoMweCWnADWwAJ4RkcVHjEA+ABQQwJMrQBccAN4AoOHFqYIMDXABKwSlARcANAZhRieAOYBtALpyA3HorNWYFoAfk0LKxtaR2cXeyinVx8-F2AYAFUSTGIYMSYWUwVDY014mIBKOABeOQcEl18AXzKwy2gbTjjoxN8dHUo8KIpqOmAEPMChSqGaCHpxtgVdfSKTTSX9OGQICE03AHIAN1RUPY8-Bttk1IziLJz5zRAqmpBLpt8gA).